src/core/ext/filters/http/server/http_server_filter.cc

```diff
@@ -206,8 +206,9 @@ static grpc_error* hs_filter_incoming_metadata(grpc_call_element* elem,
     /* offset of the character '?' */
     size_t offset = 0;
     for (offset = 0; offset < path_length && *path_ptr != k_query_separator;
-         path_ptr++, offset++)
+         path_ptr++, offset++) {
       ;
```

Comment:
Don't need this anymore with the added braces.

Reply:
Done